src/Microsoft.DotNet.XHarness.iOS.Shared/ITestReporter.cs

```diff
@@ -9,20 +9,18 @@
 
 namespace Microsoft.DotNet.XHarness.iOS.Shared
 {
-
-    // interface that represents a class that know how to parse the results from an app run.
+    /// <summary>
+    /// Interface that represents a class that knows how to parse test results.
+    /// </summary>
     public interface ITestReporter
     {
-
         ILog CallbackLog { get; }
-
```

Comment:
Noise

Reply:
I would revert it if the file didn't have any other changes and had more than 30 lines but since it's being changed already, removing a blank line doesn't hurt? Or what are your thoughts on this?